kornia/geometry/epipolar/essential.py

```diff
@@ -19,6 +23,189 @@
     "relative_camera_motion",
 ]
 
+# Reference
+# Nistér, David. An efficient solution to the five-point relative pose problem. 2004.
+# https://github.com/danini/graph-cut-ransac/blob/master/src/pygcransac/include/estimators/solver_essential_matrix_five_point_nister.h
+# Wei T, Patel Y, Matas J, Barath D. Generalized differentiable RANSAC[J]. arXiv preprint arXiv:2212.13185, 2023.
```

Comment:
you can add it as a reference link into the docstring by adding it in docs/source/reference.bib

Reply:
Thanks!